app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (
+    OutdatedVaccineError,
+    NotVaccinatedError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> str:
```

Comment:
The '__init__' method should not have a return type of 'str'. It should be 'None' since constructors in Python do not return a value.